specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "description": "A Storage Cache provides scalable caching service for NAS clients, serving data from either NFSv3 or Blob at-rest storage (referred to as \"Storage Targets\"). These operations allow you to manage Caches.",
+    "description": "A Storage Cache provides scalable caching service for NAS clients, serving data from either NFSv4 or Blob at-rest storage (referred to as \"Storage Targets\"). These operations allow you to manage Caches.",
```

Comment:
Not sure why this change shows in here. Initially I thought it was due to a merge, but I don't see any changes other than my commit that changed this